### PR TITLE
Fix using undefined variable g:ale_python_mypy_options

### PR DESCRIPTION
**Pull request by aswins**

### CODE REVIEW

File: ale_linters/python/mypy.vim [hunk 1]
```diff
@@ -6,7 +6,7 @@ let g:ale_python_mypy_args = get(g:, 'ale_python_mypy_options', '')
 function! g:ale_linters#python#mypy#GetCommand(buffer) abort
     return g:ale#util#stdin_wrapper
     \   . ' .py mypy --show-column-numbers '
-    \   . g:ale_python_mypy_options
+    \   . g:ale_python_mypy_args
```
Comment (w0rp) [inline ale_linters/python/mypy.vim]: That's the wrong fix there. Change the `let` statement above. See this: https://github.com/w0rp/ale/issues/257

Comment (aswin-zefo) [inline ale_linters/python/mypy.vim]: let g:ale_python_mypy_args = get(g:, 'ale_python_mypy_options', '')

complies with #257, right? Or do you want to change ale_python_mypy_args to ale_python_mypy_options as well?

Comment (aswins) [inline ale_linters/python/mypy.vim]: let g:ale_python_mypy_args = get(g:, 'ale_python_mypy_options', '')

complies with #257, right? Or do you want to change ale_python_mypy_args to ale_python_mypy_options as well?

Comment (w0rp) [inline ale_linters/python/mypy.vim]:
```vim
let g:ale_python_mypy_options = get(g:, 'ale_python_mypy_options', '')
```

It should be that.

**Comment by aswins** [comment]
Closing this to raise #265

**Pull request closed without merging**